lms/djangoapps/mobile_api/course_info/tests.py

```diff
@@ -116,7 +116,19 @@ def test_no_handouts(self):
         with self.store.branch_setting(ModuleStoreEnum.Branch.draft_preferred, self.course.id):
             self.store.delete_item(handouts_usage_key, self.user.id)
 
-        self.api_response(expected_response_code=404)
+        response = self.api_response(expected_response_code=200)
+        self.assertIsNone(response.data['handouts_html'])
+
+    def test_empty_handouts(self):
+        self.login_and_enroll()
+
+        # delete handouts in course but leave empty tags
```

Comment:
The comment is currently "delete handouts in course but leave empty tags"- I was trying to indicate that there are no actual handouts, but that the call to retrieve handouts will return empty tags. Would you recommend taking out the reference to deleting handouts, period?